src/igvfd/tests/fixtures/schemas/measurement_set.py

```diff
@@ -275,3 +275,17 @@ def measurement_set_with_protocols(testapp, lab, award, assay_term_starr, tissue
         'preferred_assay_title': 'SUPERSTARR'
     }
     return testapp.post_json('/measurement_set', item).json['@graph'][0]
+
+
+@pytest.fixture
+def measurement_set_with_functional_assay_mechanisms(testapp, lab, award, assay_term_starr, tissue, phenotype_term_from_go):
+    item = {
+        'award': award['@id'],
+        'lab': lab['@id'],
+        'assay_term': assay_term_starr['@id'],
```

Comment:
same as preferred_assay title, if we have assay_term for vamp-seq let's use that? Not too critical but good to have. :)

Reply:
yes same as replied in the previous comment